doc/manpages/vtysh.rst

```diff
@@ -41,6 +41,20 @@ OPTIONS available for the vtysh command:
 
    When the -c option is being used, this flag will cause the standard vtysh prompt and command to be echoed prior to displaying the results. This is particularly useful to separate the results when executing multiple commands.
 
+.. option:: -C, --dryrun
+
+   When the -C option is being used, this flag will check the config for syntatic validity.
+
+.. option:: -m, --markfile
+
+    Mark the input file with context ends, useful for cleanup of a config file
+    that has a lot of extraneous space and end markers
```

Comment:
pls use 3 space indent here and and for `-n, --noerror`

Reply:
fixed